.github/workflows/dev.backup.yml

```diff
@@ -1,7 +1,7 @@
-name: Create database backup DEV
+name: DEV - Create database backup
```

Comment:
I actually prefer the suffix than prefix

Reply:
I find that when you have multiple items in a list the prefix makes is easier to find :)